crypto/kyber/p_kyber.c

```diff
@@ -109,6 +109,108 @@ static int pkey_kyber512_decapsulate(EVP_PKEY_CTX *ctx,
   return 1;
 }
 
+static int pkey_kyber768_keygen(EVP_PKEY_CTX *ctx, EVP_PKEY *pkey) {
```

Comment:
[NP: missing return check ... we should be consistent.](https://github.com/awslabs/aws-lc/pull/637/files#r987414686)